pv_site_production/models/psp.py

```diff
@@ -11,17 +11,20 @@
 
 from pv_site_production.utils.imports import instantiate
 
+_log = logging.getLogger()
+
 
 def get_model(config: dict[str, Any], pv_data_source: PvDataSource) -> PvSiteModel:
     """Get a serialized pv-site-prediction model."""
+
+    _log.debug(f'Loading model: {config["model_path"]}')
     model = load_model(config["model_path"])
 
+    _log.debug(f'Getting NWP data: {config["nwp"]}')
     nwp_data_source = instantiate(**config["nwp"])
 
     # TODO Make the setup step uniform across all `psp` models. In other words it should be defined
     # directly in `PvSiteModel`.
-    model.setup(
-        SetupConfig(pv_data_source=pv_data_source, nwp_data_source=nwp_data_source)
-    )
+    model.setup(SetupConfig(pv_data_source=pv_data_source, nwp_data_source=nwp_data_source))
```

Comment:
I suspect your editor is not using the settings from the repo or something, if the formatting is changing here!

Reply:
i just ran blacks set to line length 100. Maybe its not set to that

Reply:
ill move it back

Reply:
Only flake8 is set to 100: https://github.com/openclimatefix/pv-site-prediction/blob/main/.flake8#L2
Black probably still has the default setting then, we should probably fix that. Doesn't matter for now I guess.